alembic/ddl/sqlite.py

```diff
@@ -60,7 +60,7 @@ def requires_recreate_in_batch(
                     return True
                 elif (
                     isinstance(col.server_default, util.sqla_compat.Computed)
-                    and col.server_default.persisted
+                    and col.server_default and col.server_default.persisted
```

Comment:
Ok, this is because of the mock we do in sqla_compat.

Thanks for reporting, but the fix should be something else.

Reply:
OK. If you fix this issue, please let me know. Thanks.